include/dali/core/util.h

```diff
@@ -84,6 +84,17 @@ constexpr Value align_up(Value v, Alignment a) {
   return v + ((a - 1) & -v);
 }
 
+template <typename Value, typename Alignment>
+DALI_HOST_DEV
+constexpr Value align_remainder(Value v, Alignment a) {
+  return v & (a - 1);
+}
```

Comment:
I'm still not sure if we need this function - and if so, I'd consider calling it `alignment_remainder` (align_remainder sounds like we want to align a remainder - whatever that means).

Reply:
Done